### DISCUSSION TITLE
允许手动将mod分类，统一禁用/开启

### DISCUSSION THREAD

**Discussion by abcdesteve** [Meloong-Git/PCL#1331] [提问] [answered]
有时候想在iris和optifine之间切换，但是两者都有前置，一个一个开启禁用很麻烦，可以手动将他们分类然后一键开启关闭整个类别里的mod吗

**Comment by WForst-Breeze** [comment, marked as answer]
不会考虑制作此功能，因为首先制作这个功能相当麻烦且适用人数较少，其次考虑到优先级过低，因此很遗憾，不会制作。
（btw，Iris 前置就一个 Sodium，Optifine 前置就一个 Optifabric，手动也不算太麻烦，从兼容性考虑，个人建议还是优先考虑性能更加出色的前者吧……毕竟高版本后者表现相对稍逊一些
